Pod/Classes/Categories/PINImage+ScaledImage.m

```diff
@@ -0,0 +1,53 @@
+//
+//  UIImage+ScaledImage.m
+//  Pods
+//
+//  Created by Michael Schneider on 2/9/17.
+//
+//
+
+#import "PINImage+ScaledImage.h"
+
+inline PINImage *PINScaledImageForKey(NSString * __nullable key, PINImage * __nullable image) {
```

Comment:
Why `inline`? You'll end up duplicating this medium-sized function body twice below, increasing the size of the executable. Seems this can be `static` without a real performance cost.